app/views/invoice_configs/_layout_fields.html.haml

```diff
@@ -0,0 +1,15 @@
+-#  Copyright (c) 2023, Puzzle ITC. This file is part of
+-#  hitobito and licensed under the Affero General Public License version 3
+-#  or later. See the COPYING file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito.
+
+= field_set_tag do
+  = f.labeled :logo do
+    = f.file_field :logo, type: :file #, accept: Group::LogoUploader.accept_extensions
```

Comment:
why is the accept: commented out?
tried to upload an gif and PDF creation failed